components/eam/src/dynamics/se/stepon.F90

```diff
@@ -171,6 +171,7 @@ subroutine stepon_run1( dtime_out, phys_state, phys_tend,               &
   use control_mod, only: ftype
   use physics_buffer, only : physics_buffer_desc
   use hycoef,      only: hyam, hybm
+  use dimensions_mod,  only: nelemd 
```

Comment:
This does not appear to be used in the changes, please remove.

Reply:
removed